tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js

```diff
@@ -58,6 +74,7 @@ const locationParams = {
         name: process.env.COLD_BACKEND_DESTINATION_LOCATION,
         supportsVersioning: true,
         isCold: true,
+        supportsRestore: true,
```

Comment:
minor: shouldn't all `isCold` location support restore?

Reply:
True. I thought this might be helpful to tune the tests in case of issues with restore or other changes.